pom.xml

```diff
@@ -138,6 +139,8 @@
 		<!-- property to skip surefire tests during failsafe execution -->
 		<!--suppress UnresolvedMavenProperty -->
 		<skip.surefire.tests>${skipTests}</skip.surefire.tests>
+		<opentelemetry-semconv.version>1.21.0-alpha</opentelemetry-semconv.version>
```

Comment:
can we move this definition together with all the others opentelemetry related above? Also it seems latest release is [1.23.1-alpha](https://github.com/open-telemetry/semantic-conventions-java/releases/tag/v1.23.1)